task_sdk/tests/execution_time/test_supervisor.py

```diff
@@ -297,7 +331,10 @@ def test_handle_requests(
         generator.send(msg)
 
         # Verify the correct client method was called
-        mock_client_method.assert_called_once_with(method_arg)
+        mock_client_method.assert_called_once_with(*method_arg)
 
+        if mock_response == "":
+            # for task instance endpoints, there won't be any response
```

Comment:
We should handle this based on how we handle the client response for task instance endpoints

Reply:
Handled it as per the client logic